curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-spread-operator-to-evaluate-arrays-in-place.english.md

```diff
@@ -45,7 +45,7 @@ tests:
   - text: <code>arr2</code> should be correct copy of <code>arr1</code>.
     testString: assert(arr2.every((v, i) => v === arr1[i]));
   - text: <code>...</code> spread operator was used to duplicate <code>arr1</code>.
-    testString: assert(code.match(/\[\s*...arr1\s*\]/g));
+    testString: assert(code.match(/\s*...arr1\s*/g));
```

Comment:
@enrider The problem with your regex is that it should not just be checking if the spread operator is used on `arr1`.  It is supposed to validate the spread operator correctly creates a copy of `arr1`.

If you read the documentation for challenge related PRs, you will find you must test the changes locally (`npm run test:curriculum`) to confirm you do not make any breaking changes.  Your change allows the user to not change any of the challenge seed code and still pass all of the tests.

Also, the original regex should not have used `...`.  Instead, it should have been `\.\.\.`.  I have made a suggestion which does pass all of the tests and correctly fails when no code is changed.  You can click `Commit suggestion` and it apply the suggested change on your behalf.

```suggestion
    testString: assert(code.match(/Array\(\s*\.\.\.arr1\s*\)|\[\s*\.\.\.arr1\s*\]/));
```